client/src/auth/auth.js

```diff
@@ -1,9 +1,21 @@
+import axios from 'axios';
+
 const auth = {
   login(token, callback) {
-    localStorage.setItem('isAuthenticated', 'true'); // default true till server is setup.
-    console.log(token);
-    callback(); // This callback is only to spoof successful login till server is setup.
-    //             Will be moved to .then later
+    const data = {
+      method: 'google',
+      access_token: token,
+    };
+    axios.post(process.env.REACT_APP_LOGIN_URL, data)
+      .then((res) => {
+        if (res.data) {
+          localStorage.setItem('isAuthenticated', 'true');
+          callback();
+        }
+      })
+      .catch(() => {
+        localStorage.setItem('isAuthenticated', 'false');
```

Comment:
Why this?
we don't have to set anything when not logged in. That shopuld be the default state aniway.

Reply:
Perhaps showing why login failed is more apt.